src/main/kotlin/com/atlassian/performance/tools/infrastructure/api/splunk/Log4jJsonifier.kt

```diff
@@ -0,0 +1,48 @@
+package com.atlassian.performance.tools.infrastructure.api.splunk
```

Comment:
Let's not leak it to the API:
```suggestion
package com.atlassian.performance.tools.infrastructure.splunk
```